components/forms/newAccount/webcam-photo.jsx

```diff
@@ -20,18 +20,22 @@ const WebCamPhoto = ({ control, isCaptureCompleted, setIsCaptureCompleted }) =>
             control={control}
             name="webcamPic"
             render={({ field }) => (
-                <FormItem>
+                <FormItem className="relative h-full">
                     <div className='flex items-center justify-between'>
-                        <FormLabel className="font-semibold">Take a webcam photo for KYC check</FormLabel>
-                        <Image src="/id_clarity.png" alt='id_clarity' width={87} height={24} />
+                        <div className='line-h-50'>
+                            <FormLabel className="font-semibold">Take a webcam photo for KYC check</FormLabel>
+                        </div>
+                        <div>
+                            <Image src="/id_clarity.png" alt='id_clarity' width={87} height={24} />
+                        </div>
                     </div>
                     <FormControl>
                         {isCaptureCompleted ? (
                             <div className='grid place-items-center'>
                                 <Image src="/example_webcam.png" alt='example_webcam' width={240} height={240} />
                             </div>
                         ) : (
-                            <div className='grid justify-items-center gap-4'>
+                            <div className='grid justify-items-center gap-4 '>
```

Comment:
Can you please remove the extra space at the end of the string?

Reply:
extra space removed, thanks for noticing.